app/src/androidTest/java/TestPlan.kt

```diff
@@ -0,0 +1,21 @@
+import androidx.test.rule.ActivityTestRule
+import com.maubis.scarlet.base.MainActivity
+import org.junit.Rule
+import org.junit.Test
+
+class TestPlan {
+
+    @Rule
+    @JvmField
+    var mActivityTestRule = ActivityTestRule(MainActivity::class.java)
+
+    @Test
+    fun openApp() {
+        Thread.sleep(2000)
```

Comment:
Magic number, can be extracted to const val

Reply:
It is not necessary. I will delete this.